tests/integration/telemetry_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"get.porter.sh/porter/pkg/porter"
+	"get.porter.sh/porter/tests/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Validate that we can configure a live connection to a telemetry endpoint
+func TestTelemetrySetup(t *testing.T) {
+	test, err := tester.NewTest(t)
+	defer test.Close()
+	require.NoError(t, err, "test setup failed")
+
+	ctx := context.Background()
+	_, _, err = test.RunPorter("install", "otel-jaeger", "-r=ghcr.io/getporter/examples/otel-jaeger:v0.1.0", "--allow-docker-host-access")
+	require.NoError(t, err)
+	defer test.RunPorter("uninstall", "otel-jaeger", "--allow-docker-host-access")
+
+	// Wait until the collection should be up
+	time.Sleep(10 * time.Second)
```

Comment:
I don't have a immediate solution in mind but this maybe flaky in some situations

Reply:
Agreed! I wanted to get _something_ to test that it's working or not for now and if I come up with a better way I'll update the test.

I tried doing a net dial but that passes just fine and then later the trace setup fails because the otel service isn't 100% ready. I feel like the best fix would be to have the porter bundle that installs the otel collection have logic and wait until the collector is up.